### ISSUE TITLE
ExpoKit SDK 34 is not compatible with latest react-native-firebase 5.5.6 due to Google-Mobile-Ads-SDK semver violation

### ISSUE THREAD

**Issue by changLiuUNSW** [expo/expo#5242]
tag ```ios/2.12.3```
Get exception:
```
Property 'customRewardString' not found on object of type 'GADRewardBasedVideoAd *'
```
The issue is caused by the library for AdMob being a lower version (Google-Mobile-Ads-SDK v7.22.0)

**Comment by changLiuUNSW** [comment]
The issue has been reported to react-native-firebase:
https://github.com/invertase/react-native-firebase/issues/2363
The workaround for now is just downgrade react-native-firebase to 5.5.5

**Comment by brentvatne** [comment]
We recommend using the bare workflow rather than ExpoKit with react-native-firebase

**Comment by ide** [comment]
Adding to Brent's note, any fixes moving forward are likely going to be for the bare workflow so I don't think we'll get around to this.

**Comment by changLiuUNSW** [comment]
@ide I agree, but I don’t think OTA is available in the bare workflow ?

**Comment by ide** [comment]
That's correct for right now. We are working on it as well as a high-efficiency updates module (maybe for next year). Either way, our energy is dedicated to the bare workflow as we build replacements (e.g. updates) for ExpoKit to phase it out.

**Comment by mikehardy** [comment]
@ide just wanted to mention that on further review in the linked react-native-firebase issue it seems that when iOS 13 rolls out people are going to be stuck in a tangle that they can't fix without an ExpoKit dependency update.

Specifically:
- firebase-ios-sdk <= 6.3.0 has a crash bug with iOS 13 that is fixed maybe with 6.4.0 and definitely with 6.5.0 (6.6.0 is current as of this writing
- Firebase/AdMob 6.5.0 and higher depend on Google-Mobile-Ads-SDK 7.44
- ExpoKit 33 (and 34 I suppose?) seem to depend on 7.22

There may be more subtlety to which versions of who can tolerate which ranges etc but my point is:

if ExpoKit 34 (or a patch release or something?) could bump the underlying Google-Mobile-Ads-SDK to the current version, that would allow folks to move to current versions all around (react-native-firebase current, firebase-ios-sdk current) and avoid a crash bug 

I have no idea the complexity of this on the ExpoKit side but as it's a looming crash bug triggered by an iOS release that's coming, I thought it worth a mention at least. Cheers!

**Comment by ide** [comment]
Ok. We'll take a look but if there's any progress on this front, it will likely be for SDK 35.

**Comment by lenichols** [comment]
Having a ton of issues with FIrebase in Expokit. Hopefully this update is released soon.
